src/firefly/js/visualize/task/PlotImageTask.js

```diff
@@ -398,3 +422,37 @@ function initBuildInDrawLayers() {
     DrawLayerCntlr.dispatchCreateDrawLayer(ActiveTarget.TYPE_ID);
 }
 
+/**
+ *
+ * @param {String[]} plotIdAry
+ * @param {Object.<string, OverlayPlotView>} oldOverlayPlotViews
```

Comment:
Is it one OverlayPlotView or an array? enableMatchingRelatedData seems to expect an array.